src/orquestra/sdk/_base/_testing/_example_wfs.py

```diff
@@ -194,6 +194,23 @@ def add_with_trigger(a, b, port, timeout: float):
     return a + b
 
 
+@sdk.task
+def infinite_task(*_):
+    import time
+
+    while True:
```

Comment:
**suggestion**: could you add a timeout for this loop? We probably don't want to hang forever if something goes wrong

Reply:
This is ray worker that is supposed to be terminated. Worst case scenario you have to restart ray. It doesnt block - thus I think it should be actually <infinite>

Reply:
But ill add like 1hour timeout or so